### PR TITLE
Finish porting AWS to Jinja2

### PR DESCRIPTION
**Pull request by lerks**
See #886 for the first part.

---
This change is [Reviewable](https://reviewable.io/reviews/cms-dev/cms/900)

### CODE REVIEW

File: cms/server/admin/handlers/base.py [hunk 1]
```diff
@@ -305,6 +305,9 @@ def render_params(self):
         params["contest"] = self.contest
         params["url"] = self.url
         params["xsrf_form_html"] = self.xsrf_form_html()
+        # FIXME these are cheating
+        params["config"] = config
+        params["handler"] = self
```
Comment (stefano-maggiolo) [inline cms/server/admin/handlers/base.py]: Is it me or the commit title doesn't agree much with what the commit does?

Comment (lerks) [reply cms/server/admin/handlers/base.py]: The first line of the commit message is correct (config and handler are values that, under Tornado, were either automatically injected by Tornado or imported by templates). But, yes, the rest of the message has nothing to do with it and appears to be duplicated from the following commit. Not sure how that happened.

File: cms/server/util.py [hunk 2]
```diff
@@ -375,29 +375,57 @@ def get_url_root(request_path):
         return "."
 
 
-def create_url_builder(url_root):
-    """Return a function that builds an URL relative to the given root.
+class Url(object):
```
[the remark this answers is not in the document]
Comment (lerks) [reply cms/server/util.py]: Done

**Comment by stefano-maggiolo** [comment]
LGTM

**Comment by lerks** [comment]
All pushed to master!

**Pull request closed without merging**